### DISCUSSION TITLE
v0.11.0

### DISCUSSION THREAD

**Discussion by ChristopherHX** [minecraft-linux/mcpelauncher-manifest#878] [Announcements]
- Resolve google login problems (error "Login failed") of v0.10.0 and older due to missing accountIdentifier caused by server side changes from Google
- Resolve compatibility problems with 1.20.20 and 1.20.30
- Basic Gamepad Navigation via dpad (l/r => backtab/tab, up/down scroll up/down) and a/b
- Gamepad configuration tool and notification about missing gamepad Mappings (notifications can be hidden via a new setting)

<hr /><em>This discussion was created from the release <a href='https://github.com/minecraft-linux/mcpelauncher-manifest/releases/tag/v0.11.0'>v0.11.0</a>.</em>

**Comment by kjjjjjjjjjjjj** [comment]
Can I join with your project

**Comment by kjjjjjjjjjjjj** [comment]
As a creator

**Reply by ChristopherHX** [reply]
Sorry but I have no idea who you are, until then you are free to open Pull Requests and I will have a look at your changes.

If you are a useful and reliable contributor for a longer timespan you may be able to gain a higher access level to some of the repositories.

Sorry, but your display name in the email
> x minecaaft launcher

Looks like scam

**Reply by kjjjjjjjjjjjj** [reply]
Oh okay
